app/lib/tool/test_profile/models.dart

```diff
@@ -59,6 +59,7 @@ class TestPackage {
   final String? replacedBy;
   final bool? isUnlisted;
   final bool? isFlutterFavorite;
+  final List<String>? retractedVersions;
```

Comment:
I think this is a simple and good enough solution for now, but I'd imagine if/when we'll have more flags for specific versions, it should go inside the `versions:` block.